quickstart.html

```diff
@@ -34,12 +34,12 @@ <h4>Setting up browser integration with KeePassXC</h4>
     <ul>
         <li><em>Within KeePassXC</em>, go to <strong>Tools-&gt;Settings</strong>
             (on macOS, go to <strong>KeePassXC-&gt;Preferences</strong>).
-        <li>In <strong>Browser Integration</strong>, check <strong>Enable KeePassHTTP server</strong>
+        <li>In <strong>Browser Integration</strong>, check <strong>Enable KeePassXC browser integration</strong>
             Leave the other options at their defaults.
```

Comment:
Instead of "Leave the other options at their defaults", we need to tell users to check the box for their browser, possibly with screenshot.

Reply:
Screenshot and explanation added.